platform/linux-generic/odp_schedule_scalable.c

```diff
@@ -1025,9 +1027,23 @@ static void schedule_order_unlock(unsigned lock_index)
 	atomic_store_release(&rctx->rwin->olock[lock_index],
 			     rctx->sn + 1,
 			     /*readonly=*/false);
+	rctx->rwin->lock_index = rctx->rwin->lock_count + 1;
```

Comment:
Delete, as per previous explanation. The scalable scheduler could be further optimized by changing its runtime checks to ODP_ASSERT()s, but that's beyond the scope of this PR.